src/tiled/commandmanager.h

```diff
@@ -39,13 +40,22 @@ class CommandManager : public QObject
 
     static void deleteInstance();
 
+    /**
+     * Returns the CommandDataModel instance stored
+     */
     CommandDataModel *commandDataModel();
 
+    /**
+     * Registers a new QMenu with the CommandManager
+     */
+    void registerMenu(QMenu* menu);
+
 public slots:
+
     /**
      * Populates the menu pointed by menu
      */
-    void populateMenu(QMenu *menu, bool flag = false);
+    void populateMenu();
```

Comment:
Since this populates all registered menus, it should be called `populateMenus` now. You also still need to update the documentation.